net.studio08.xbplay.yml

```diff
@@ -0,0 +1,63 @@
+app-id: net.studio08.xbplay
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: net.studio08.xbplay
+    buildsystem: simple
+    build-commands:
+      # setup build files
+      - cp -a . /app/main
+      - chmod +x /app/main/com.studio08.xbplay
+
+      # add icons
+      - install -Dm644 icon_32.png /app/share/icons/hicolor/32x32/apps/net.studio08.xbplay.png
+      - install -Dm644 icon_64.png /app/share/icons/hicolor/64x64/apps/net.studio08.xbplay.png
+      - install -Dm644 icon_128.png /app/share/icons/hicolor/128x128/apps/net.studio08.xbplay.png
+      - install -Dm644 icon_256.png /app/share/icons/hicolor/256x256/apps/net.studio08.xbplay.png
```

Comment:
```suggestion
```

Reply:
Is there a reason I shouldn't include these icons?